backend/alembic/versions/0024_sibling_roms_db_view.py

```diff
@@ -0,0 +1,57 @@
+"""empty message
+
+Revision ID: 0024_sibling_roms_db_view
+Revises: 0023_make_columns_non_nullable
+Create Date: 2024-08-08 12:00:00.000000
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "0024_sibling_roms_db_view"
+down_revision = "0023_make_columns_non_nullable"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    connection = op.get_bind()
+
+    connection.execute(
+        sa.text(
+            """
+            CREATE VIEW sibling_roms AS
+            SELECT
+                r1.id AS rom_id,
+                r2.id AS sibling_rom_id,
+                NOW() AS created_at,
+                NOW() AS updated_at
+            FROM
+                roms r1
+            LEFT JOIN 
```

Comment:
Should this be a regular JOIN instead, to avoid rows where sibling_rom_id is null?